lib/analyzer/static-analyzer.ts

```diff
@@ -108,9 +115,11 @@ export async function analyze(
     staticAnalysisActions.push(
       ...[
         getNodeAppFileContentAction,
+        getNodeJsTsAppFileContentAction,
         getPhpAppFileContentAction,
         getPoetryAppFileContentAction,
         getPipAppFileContentAction,
+        getPythonAppFileContentAction,
         getJarFileContentAction,
         getGoModulesContentAction,
       ],
```

Comment:
push getNodeJsTsAppFileContentAction and getPythonAppFileContentAction only if collectApplicationFiles